mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -120,7 +120,7 @@ CLASS net/minecraft/unmapped/C_usxaxydn net/minecraft/entity/LivingEntity
 	METHOD m_awmtskvv getMovementSpeed (F)F
 		ARG 1 slipperiness
 	METHOD m_axkhmdzu getDeathSound ()Lnet/minecraft/unmapped/C_avavozay;
-	METHOD m_ayeajhdm setCurrentHand (Lnet/minecraft/unmapped/C_laxmzoqs;)V
+	METHOD m_ayeajhdm startUsing (Lnet/minecraft/unmapped/C_laxmzoqs;)V
```

Comment:
```suggestion
	METHOD m_ayeajhdm tryStartUsing (Lnet/minecraft/unmapped/C_laxmzoqs;)V
```
i would prefix with `try`, since it won't start using unless there's a valid item

Reply:
good point, I'll add a javadoc, too